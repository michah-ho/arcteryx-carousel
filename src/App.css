.container{
  padding:0 60px;
}
@media screen and (max-width:768px){
  .container{
    padding:0 15px;
  }
}
.text-center{
  text-align:center;
}
/*Slick Slider Overrides*/
.slick-prev:before, .slick-next:before {
  color: #000;
}

.slick-track {
  display: flex !important;
}
.slick-slide {
  height: auto;
  display: flex !important;
  flex-direction: column;
  justify-content: center;
  padding:0 30px;
}
.slick-slide img{
  max-width:100%;
}
