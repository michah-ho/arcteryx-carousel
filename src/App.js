import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Arcteryx Carousel</h1>
    </div>
  );
}

export default App;
