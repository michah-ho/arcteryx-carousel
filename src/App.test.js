import { render, screen } from '@testing-library/react';
import App from './App';
import {shallow} from 'enzyme';



describe("Does App Render correctly", () =>{
  
  const wrapper = shallow(<App/>);

  it("should render without crashing", ()=>{
    const mockResponse = { img_url: "http://fake.com/donuts.jpg" };
    fetch.mockResponseOnce(JSON.stringify(mockResponse));
    expect(wrapper).toMatchSnapshot();
  })
})
