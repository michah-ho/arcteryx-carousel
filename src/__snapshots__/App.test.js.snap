// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does App Render correctly should render without crashing 1`] = `
<div
  className="App"
>
  <h1>
    Arcteryx Carousel
  </h1>
  <SlickSlider
    imageList={
      Object {
        "data": Object {},
        "loading": false,
      }
    }
  />
</div>
`;
